database.py

```diff
@@ -0,0 +1,62 @@
+import pymongo
+import os
+import re
+from dotenv import load_dotenv
+
+
+class myDb:
+    """
+    A class used to represent the MongoDB
+
+    Attributes
+    ----------
+    client: pymongo.MongoClient
+        A reference to the MongoDB client.
+    uri: string
+        The URL used for the MongoDB driver. This will be stored on the Heroku for prod and in an .env file for test.
+    db_name: string
+        The name of the database we are using.
+
+    Methods
+    -------
+    get_db_uri()
+        Returns the MongoDB URL.
+
+    get_db_client()
+        Returns the active MongoClient.
+
+    get_db_collection(collection_name)
+        Returns a reference to the specifield collection in the active client.
+
+    get_one(collection, query)
+        Returns a cursor containing all the items from the collection that match the query.
+    """
+
+    client = None
+    uri = None
+    db_name = None
+
+    def __init__(self, uri=os.environ["MONGODB_URI"]):
+        if self.uri == None:
+            load_dotenv()
+            self.uri = os.getenv("MONGODB_URI")
```

Comment:
Why don't we just have `uri=None` as the default?  It'll fall into the `self.uri == None` case anyway, and get set to $MONGODB_URI that way.

Reply:
Ah yes I have realised that `os.environ()` & `os.getenv()` are virtually the same. I'll make the change!